articles/changes.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Foreword
+sidebar_label: 2023 Changes
+---
+
+export function Card() {
+  return (
+      <div className="flex mb-10">
+        <div
+            className="flex items-center justify-center">
+          <img className="!h-20 w-20 ring ring-brand-50 ring-offset-2 !rounded-full mr-6" src='/img/alex.png' alt=""/>
+          <div className="flex-1 gap-1">
+            <div className="text-xl font-medium text-brand pb-0!">Alex Godbehere</div>
+            <div className="text-sm truncate text-brand-80">Technical Fellow for Smart Factories</div>
+            <a href={'mailto:alex.godbehere@amrc.co.uk'} className="text-sm text-brand-80 truncate">alex.godbehere@amrc.co.uk</a>
+          </div>
+        </div>
+      </div>
+  )
+}
+
+<Card></Card>
+
+Factory+ was ultimately developed to be a vehicle to help solve manufacturing problems. A project that started out as a small piece of research into current best practices for digital factory architectures quickly evolved into a robust framework that is now adopted as an operational standard across the AMRC and beyond. Because Factory+ defines a scalable, robust approach to architecture, data models, transport protocols, storage strategies, contextualisation and consumption of manufacturing data, we as a research organisation can focus on the things that matter - using the data that we're collecting to impart meaningful changes in the manufacturing sector.
```

Comment:
```suggestion
Factory+ was ultimately developed to be a vehicle to help solve manufacturing problems. A project that started out as a small piece of research into current best practices for digital factory architectures quickly evolved into a robust framework that is now adopted as an operational standard across the [AMRC](https://www.amrc.co.uk/), the UK's largest High Value Manufacturing Catapult Centre, and beyond. Because Factory+ defines a scalable, robust approach to architecture, data models, transport protocols, storage strategies, contextualisation and consumption of manufacturing data, we as a research organisation can focus on the things that matter - using the data that we're collecting to impart meaningful changes in the manufacturing sector.
```